aiida_mlip/workflows/hts_workgraph.py

```diff
@@ -0,0 +1,64 @@
+""" Workgraph to run DFT calculations and use the outputs fpr training a MLIP model."""
```

Comment:
```suggestion
"""WorkGraph to run DFT calculations and use the outputs for training a MLIP model."""
```
We probably want a file name that distinguishes this from hts.py better too